clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/security/OpenstackNamedAccountCredentials.groovy

```diff
@@ -201,7 +201,7 @@ class OpenstackNamedAccountCredentials implements AccountCredentials<OpenstackCr
       return this
     }
 
-    private OpenstackCredentials build() {
+    OpenstackCredentials build() {
```

Comment:
This is package-private now, you probably want `public OpenstackCredentials build()`